pdfgen.c

```diff
@@ -2804,6 +2852,11 @@ int pdf_add_image_data(struct pdf_doc *pdf, struct pdf_object *page, float x,
                        float y, float display_width, float display_height,
                        const uint8_t *data, size_t len)
 {
+    if (display_width < 0 && display_height < 0) {
```

Comment:
Is this still necessary? Won't this be caught by `get_img_display_dimensions`?

Reply:
Yeah this check is redundant now, I'll remove it.